java/src/main/java/org/rocksdb/SstFileWriter.java

```diff
@@ -13,9 +13,6 @@
  * sequence number = 0.
  */
 public class SstFileWriter extends RocksObject {
```

Comment:
How is the library loaded for SStFileReader/Writer?

Reply:
I believe the idea is that the `SstFileWriter/Reader` ctors take some `Options`, which in turn ensure that the library is already loaded.

Reply:
@jaykorean It's exactly as @ltamasi mentioned. First you need to create `Options` which would load the library. With this PR I tried to minimise where we check for loaded library as it may affect performance.